### PR TITLE
Milestone : Leader Board Setup

### PR DESCRIPTION
**Pull request by Gleennkar**
## Setup my Leader Board Project
- Setup webpack
- Used gitflow
- Updated my README.md
- Run Linter Checks
- Add index.html and style.css file
- Used ES6 best practices

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <section>
+        <h2>Add scores</h2>
+        <form>
+            <input type="text" id="name" placeholder="Your name" ><br>
+            <input type="number" id="score" placeholder="Your score"><br>
+            <input type="submit" id="submit" value="Submit">
+        </form>
+    </section>
+</body>
+</html>
```
Comment (julie-ify) [inline dist/index.html]: - [ ] Kindly make your project match the given wireframe and have the placeholder scores too.

Your Implementation | Wireframe
:---:|:---:
 |

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Gleennkar,

Good job so far!

## Highlights!
- Gitflow ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (Whoistolu) [reply dist/index.html]: - [ ] [REQUIRED] Great job implementing the recommended changes, however, you still need to make some adjustments to ensure it looks exactly like the provided wireframe by separating the input fields of the name and score from the scoreboard as it is [here](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png).
### Your Current Implementation
Screenshot from 2022-09-01 22-13-44

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Gleennkar 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Gleennkar 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**